devices/generic/items/state_music_sync_effect_item.json

```diff
@@ -0,0 +1,8 @@
+{
+  "schema": "resourceitem1.schema.json",
+  "id": "state/music_sync_effect",
```

Comment:
I would prefer to group the `state/music_sync` attributes, so this would be renamed to `state/music_sync/effect`.  We would also need to change `state/music_sync` to `state/music_sync/on`.

I think the values should be reported as strings in the API, and we need a `cap` item to report the possible values.